### PR TITLE
CEDS-1907 Align model with backend

### PR DESCRIPTION
**Pull request by PatrykRudnicki**

### CODE REVIEW

File: app/controllers/declaration/DestinationCountriesController.scala [hunk 1]
```diff
@@ -50,139 +47,89 @@ class DestinationCountriesController @Inject()(
...
-              destinationCountriesStandardPage(
-                mode,
-                Standard.form(data.countryOfDispatch, data.countryOfDestination).fill(data),
-                data.countriesOfRouting
-              )
-            )
-        }
-      case None => Redirect(controllers.declaration.routes.OriginationCountryController.displayPage(mode))
+    request.declarationType match {
+      case DeclarationType.SUPPLEMENTARY => Redirect(routes.DestinationCountryController.displayPage())
```
Comment (ed0906) [inline app/controllers/declaration/DestinationCountriesController.scala]: Is this an infinite loop?

Comment (ed0906) [reply app/controllers/declaration/DestinationCountriesController.scala]: We probably need a `ReturnToStartException` for this kind of thing

Comment (PatrykRudnicki) [reply app/controllers/declaration/DestinationCountriesController.scala]: `DestinationCountryController` is different controller. I know stupid naming, but this Controller will disappear in next PR, so I decided to don't refactor this. Whole this controller now support just one input `countriesOfRouting` and will be changed and renamed

File: app/models/declaration/Locations.scala [hunk 2]
```diff
@@ -23,17 +23,19 @@ import models.ExportsDeclaration
 import play.api.libs.json.Json
 
 case class Locations(
-  destinationCountries: Option[DestinationCountries] = None,
+  originationCountry: Option[String] = None,
```
Comment (ed0906) [inline app/models/declaration/Locations.scala]: `origination` is not a word.... :p

**Pull request merged**